src/lib/swapper/swappers/ZrxSwapper/endpoints.ts

```diff
@@ -49,16 +50,24 @@ export const zrxApi: SwapperApi = {
 
     if (zrxQuoteResponse.isErr()) throw zrxQuoteResponse.unwrapErr()
 
-    const { value, to, gasPrice, gas, data } = zrxQuoteResponse.unwrap()
+    const { value, to, data } = zrxQuoteResponse.unwrap()
+
+    const feeData = await getFees({
+      adapter: assertGetEvmChainAdapter(chainId),
+      data: data.toString(),
+      to,
+      value: value.toString(),
```

Comment:
This should already be a string according to types

```suggestion
      value,
```